bpf/types.h

```diff
@@ -212,10 +212,16 @@ struct filter_value_t {
     u8 protocol;
     u16 dstPortStart;
     u16 dstPortEnd;
+    u16 dstPort1;
+    u16 dstPort2;
     u16 srcPortStart;
     u16 srcPortEnd;
+    u16 srcPort1;
+    u16 srcPort2;
     u16 portStart;
     u16 portEnd;
+    u16 port1;
+    u16 port2;
```

Comment:
Why not having more than two values here ? 
Is it just a matter of keeping the code simple here or are there other limitations ?

Reply:
no we can have more but the question is how many more ? it can't be variable size or dynamic so I just limited to two for now per rule

Reply:
that's something we should define according to the use cases we want to cover :smile_cat: 

cc @stleerh @jotak

Reply:
I guess if someone wants a third one, we can move to a fixed size array such as 8 ? (next time)